app/src/main/java/com/example/notes2text/adapters/DirectoryAdapters/ActivitySwitchController.java

```diff
@@ -43,23 +44,30 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(binding.getRoot());
         //get the path from the intent that passed to this activity and bundle it.
         initialPath = getIntent().getStringExtra("path");
-        ArrayList<File> initialFileList = new ArrayList<File>();
+        ArrayList<File> initialFileList = new ArrayList<>();
         try{
         Bundle bundle = getIntent().getExtras();
         initialFileList = (ArrayList<File>) bundle.getSerializable("selectedFiles");
         } catch (ClassCastException e){
-            initialFileList = new ArrayList<File>();
+            initialFileList = new ArrayList<>();
         }
         if (initialFileList == null){
             DirectoryAccessController dirAcCntrl = DirectoryAccessController.newInstance(initialPath);
             replaceFragment(dirAcCntrl);
+            Log.i("switch", "directory controller");
         } else if (initialFileList.isEmpty()) {
             SelectionController sltnCntrl = SelectionController.newInstance(initialPath);
             replaceFragment(sltnCntrl);
+            Log.i("switch", "selection controller");
         } else{
             SelectionController sltnCntrl = SelectionController.newInstance(initialPath, initialFileList);
             replaceFragment(sltnCntrl);
             Toast.makeText(getApplicationContext(), "ActivitySwitchController received files", Toast.LENGTH_SHORT).show();
+            try{
+            Log.i("switch", "selection controller with files");
+            } catch (Exception e){
+                Log.i("inherit files", "file names unavailable");
+            }
```

Comment:
Seems to not be doing anything, consider removing this try and catch

Reply:
Fixed in next commit, thanks.